modules/codemod/lib/v8/spec/promoteBreadcrumbs.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('Rename Box, Flex and Stack', () => {
 
     it('should not transform other preview imports', () => {
       const input = stripIndent`
-          import { Select } from "@workday/canvas-kit-preview-react/select";
```

Comment:
this is a v8 codemod, I don't think we want to touch this file

Reply:
I think I was searching for old `preview-react/select` which I shouldnt have.

Reply:
been there done that 😂